### DISCUSSION TITLE
Question: Add team for CODEOWNERS from other organization

### DISCUSSION THREAD

**Discussion by mszostok** [community/community#21781] [New to GitHub]
<p>Hi,</p>
<p>In documentation there is information that teams can be assigned in CODEOWNERS file.</p>
<pre><code>(..) or team names using the standard @username or @org/team-name.
</code></pre>
<p>as we can see there is format <span class="mention">@org</span>/team-name. That means that I can assign team from different organization?</p>
<p>Example, org Foo has only team definition:</p>
<p><strong>Org:</strong> Foo</p>
<p><strong>Teams:</strong> A,B,C</p>
<hr>
<p>Org Bar has only repos with teams codeowners from Foo organization</p>
<p><strong>Org:</strong> Bar</p>
<p><strong>Repository: </strong> XYZ: with CODEOWNERS:</p>
<pre><code># Files owned by team from Foo organization
*.go @Foo/A
</code></pre>
<p>Thanks in advance for your reply :) </p>
<p>Cheers,</p>
<p>Mateusz</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Thank you <a class="mention" href="/mszostok">@mszostok</a> for being here and for your excellent question!  Although <span class="mention">@org</span>/team-name is apart of the permitted syntax for adding CODEOWNERS, <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-team-access-to-an-organization-repository" rel="nofollow noopener">teams are per organization</a> and code owners are per repository.  So there is presently no cross-over in allowing CODEOWNER access from one organization’s team to another organization using that assigning syntax.</p>
<p>You would need to configure a team per organization to encompass the users that you wish to have as <a href="https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners" rel="nofollow noopener">CODEOWNER</a> per repository.</p>

**Reply by dominictobias-bullish** [reply]
And for any file I assume it's just asterisk `*`?

**Comment by justaverylee** [comment]
I too would love to have this feature. It was also asked for in https://github.com/isaacs/github/issues/953

**Reply by macetw** [reply]
Same. Particularly in a github-enterprise context.

**Comment by tvsaru** [comment]
4years later, I encountered the same confusion on what the syntax says is possible:

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
```
# Teams can be specified as code owners as well. Teams should
# be identified in the format @org/team-name. Teams must have
# explicit write access to the repository. In this example,
# the octocats team in the octo-org organization owns all .txt files.
*.txt @octo-org/octocats
```

Vs what was explained here - Teams from different orgs can't be mentioned in CODEOWNERS.

May be it is time to update the doc to explicitly call that out...

**Comment by baelter** [comment]
@nyahbhinghiPrincess It would make sense to allow this for enterprise accounts with many orgs.

**Comment by afrazkhan** [comment]
Another 4 years and it's the same story. I wonder what the point of requiring the `@org/` syntax was in the first place?
